src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/MethodologySummary.test.tsx

```diff
@@ -424,14 +434,90 @@ describe('MethodologySummary', () => {
     );
   });
 
+  describe('Removing a non-amendment methodology', () => {
+    test('the remove methodology button is shown if user has permission', () => {
+      render(
+        <MemoryRouter>
+          <MethodologySummary
+            publication={testPublicationWithMethodologyCanRemove}
+            topicId={testTopicId}
+            onChangePublication={noop}
+          />
+        </MemoryRouter>,
+      );
+
+      expect(
+        screen.queryByText('Remove', { selector: 'button' }),
```

Comment:
General recommendation would be to use `getByRole('button', { name: 'Remove' });` for buttons.

The main advantage of using this is that the error output for `getByRole` explains what the accessible elements are (such as what buttons are actually on the page).

Reply:
Gotcha - have switched them all over, and also answered one of Amy's old mysteries as to why she could get one to work but not the other, being all down to visibility!  Thanks.